packages/ui/src/components/UnitInput/__stories__/State.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import type { StoryFn } from '@storybook/react'
+import { UnitInput } from '..'
+import { Stack } from '../../Stack'
+import { Template } from './Template.stories'
+
+const optionsSelect = [
+  {
+    label: 'Seconds',
+    value: 'seconds',
+  },
+  {
+    label: 'Days',
+    value: 'days',
+  },
+  {
+    label: 'Months',
+    value: 'months',
+  },
+]
+export const States: StoryFn<typeof UnitInput> = () => (
+  <Stack gap={2}>
+    <UnitInput
+      options={optionsSelect}
+      success="success"
+      label="Success"
+      name="Success"
+    />
+    <UnitInput
+      options={optionsSelect}
+      error="error"
+      label="Error"
+      name="Error"
+    />
+  </Stack>
+)
```

Comment:
You need to get the props and spread it in components or storybook won't add code snippet for some reasons:
![Screenshot 2024-05-13 at 15 48 27](https://github.com/scaleway/ultraviolet/assets/15812968/3efb2f8c-5c0d-49a3-9fb1-2a7797a1be6e)

```suggestion
export const States: StoryFn<typeof UnitInput> = (props) => (
  <Stack gap={2}>
    <UnitInput
      options={optionsSelect}
      success="success"
      label="Success"
      name="Success"
      {...props}
    />
    <UnitInput
      options={optionsSelect}
      error="error"
      label="Error"
      name="Error"
      {...props}
    />
  </Stack>
)
```